NvCodecCliOptions.cpp:1306:30: error: invalid conversion from 'int' to 'NV_ENC_VUI_COLOR_PRIMARIES {aka _NV_ENC_VUI_COLOR_PRIMARIES}

**Describe the bug**
In file included from /code/VideoProcessingFramework/PyNvCodec/TC/inc/MemoryInterfaces.hpp:19:0,
                 from /code/VideoProcessingFramework/PyNvCodec/TC/src/NvCodecCliOptions.cpp:20:
/code/Video_Codec_SDK_12.0.16/Interface/nvEncodeAPI.h:242:37: note: declared here
 NV_ENC_DEPRECATED static const GUID NV_ENC_PRESET_LOSSLESS_HP_GUID =
                                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/code/VideoProcessingFramework/PyNvCodec/TC/src/NvCodecCliOptions.cpp: In member function 'void VPF::NvEncoderClInterface::SetupVuiConfig(NV_ENC_CONFIG_H264_VUI_PARAMETERS&, VPF::ParentParams&, bool, bool) const':
/code/VideoProcessingFramework/PyNvCodec/TC/src/NvCodecCliOptions.cpp:1305:26: error: invalid conversion from 'int' to 'NV_ENC_VUI_VIDEO_FORMAT {aka _NV_ENC_VUI_VIDEO_FORMAT}' [-fpermissive]
     params.videoFormat = 5;
                          ^
/code/VideoProcessingFramework/PyNvCodec/TC/src/NvCodecCliOptions.cpp:1306:30: error: invalid conversion from 'int' to 'NV_ENC_VUI_COLOR_PRIMARIES {aka _NV_ENC_VUI_COLOR_PRIMARIES}' [-fpermissive]
     params.colourPrimaries = 2;
                              ^
/code/VideoProcessingFramework/PyNvCodec/TC/src/NvCodecCliOptions.cpp:1307:38: error: invalid conversion from 'int' to 'NV_ENC_VUI_TRANSFER_CHARACTERISTIC {aka _NV_ENC_VUI_TRANSFER_CHARACTERISTIC}' [-fpermissive]
     params.transferCharacteristics = 2;
                                      ^
/code/VideoProcessingFramework/PyNvCodec/TC/src/NvCodecCliOptions.cpp:1308:27: error: invalid conversion from 'int' to 'NV_ENC_VUI_MATRIX_COEFFS {aka _NV_ENC_VUI_MATRIX_COEFFS}' [-fpermissive]
     params.colourMatrix = 2;
                           ^
PyNvCodec/TC/CMakeFiles/TC.dir/build.make:187: recipe for target 'PyNvCodec/TC/CMakeFiles/TC.dir/src/NvCodecCliOptions.cpp.o' failed
make[2]: *** [PyNvCodec/TC/CMakeFiles/TC.dir/src/NvCodecCliOptions.cpp.o] Error 1
CMakeFiles/Makefile2:347: recipe for target 'PyNvCodec/TC/CMakeFiles/TC.dir/all' failed
make[1]: *** [PyNvCodec/TC/CMakeFiles/TC.dir/all] Error 2
Makefile:135: recipe for target 'all' failed
make: *** [all] Error 2


**Expected behavior**
A clear and concise description of what you expected to happen.


**Desktop (please complete the following information):**
 - OS: ubuntu18.04
 - Nvidia driver version 515.65.01
 - CUDA Version [ 10.2]
 - Video Codec SDK Version [12.0.16]
 - Python Version [ 3.7]

**Additional context**
Add any other context about the problem here.

--- a/PyNvCodec/TC/src/NvCodecCliOptions.cpp
+++ b/PyNvCodec/TC/src/NvCodecCliOptions.cpp
@@ -1302,10 +1302,10 @@ void NvEncoderClInterface::SetupVuiConfig(
   if (!is_reconfigure) {
     memset(&params, 0, sizeof(params));
 
-    params.videoFormat = 5;
-    params.colourPrimaries = 2;
-    params.transferCharacteristics = 2;
-    params.colourMatrix = 2;
+    params.videoFormat = NV_ENC_VUI_VIDEO_FORMAT_UNSPECIFIED;
+    params.colourPrimaries = NV_ENC_VUI_COLOR_PRIMARIES_UNSPECIFIED;
+    params.transferCharacteristics = NV_ENC_VUI_TRANSFER_CHARACTERISTIC_UNSPECIFIED;
+    params.colourMatrix = NV_ENC_VUI_MATRIX_COEFFS_UNSPECIFIED;
   }
 
   if (print_settings) {

--- a/PyNvCodec/src/PyFFMpegDecoder.cpp
+++ b/PyNvCodec/src/PyFFMpegDecoder.cpp
@@ -123,7 +123,7 @@ py::array_t<MotionVector> PyFfmpegDecoder::GetMotionVectors()
   size /= sizeof(*ptr);
 
   if (ptr && size) {
-    py::array_t<MotionVector> mv({size});
+    py::array_t<MotionVector> mv({static_cast<int64_t>(size)});
     auto req = mv.request(true);
     auto mvc = static_cast<MotionVector*>(req.ptr);
 

